css/templates/travel/page-vars.css

```diff
@@ -112,3 +137,95 @@
   @custom-media --breakpoint-md-lg (min-width: 52.06rem) and (max-width: 72rem);
   @custom-media --breakpoint-v-md-lg (min-height: 52.06rem) and (max-height: 72rem);
 }
+
+body {
+  color: var(--font-color-primary);
+  font-family: var(--font-family-primary);
+  line-height: var(--line-height-2);
+  box-sizing: border-box;
+  margin: 0;
+  -webkit-font-smoothing: antialiased;
+}
+
+*, *::before, *::after {
+  box-sizing: inherit;
```

Comment:
drop the * its already there in - https://github.com/ampproject/ampstart/blob/master/css/ampstart-base/base-page.css#L19